src/wallet.cpp

```diff
@@ -1787,7 +1787,7 @@ bool CWallet::GetStakeWeight(uint64_t& nWeight)
     int64_t nCurrentTime = GetAdjustedTime();
     CTxDB txdb("r");
     //Retrieve CPID RSA_WEIGHT
-    int64_t RSA_WEIGHT = GetRSAWeightByCPID(GlobalCPUMiningCPID.cpid);
+    int64_t RSA_WEIGHT = IsV8Enabled(nBestHeight+1) ? 0 : GetRSAWeightByCPID(GlobalCPUMiningCPID.cpid);
```

Comment:
i think better actual `GetRSAWeightByCPID` should be changed

Reply:
i put it here because the newbie boost is no more and it's likely that GetRSAWeightByCPID could be removed in future.